internal/appsec/listener/waf/waf.go

```diff
@@ -87,16 +87,23 @@ func (waf *Feature) onStart(op *waf.ContextOperation, _ waf.ContextArgs) {
 	waf.SetupActionHandlers(op)
 }
 
-func (waf *Feature) SetupActionHandlers(op *waf.ContextOperation) {
+func (*Feature) SetupActionHandlers(op *waf.ContextOperation) {
 	// Set the blocking tag on the operation when a blocking event is received
 	dyngo.OnData(op, func(_ *events.BlockingSecurityEvent) {
+		log.Debug("appsec: blocking event detected")
 		op.SetTag(BlockedRequestTag, true)
 	})
 
 	// Register the stacktrace if one is requested by a WAF action
 	dyngo.OnData(op, func(err *actions.StackTraceAction) {
+		log.Debug("appsec: registering stack trace for security purposes")
 		op.AddStackTraces(err.Event)
 	})
+
+	dyngo.OnData(op, func(_ *waf.SecurityEvent) {
```

Comment:
```suggestion
	dyngo.OnData(op, func(*waf.SecurityEvent) {
```

Reply:
Applied in https://github.com/DataDog/dd-trace-go/pull/3033/commits/2be912cbe48a65d29cbee2f0699b41ef8e34e0bc